cim-forward-route/src/main/java/com/crossoverjie/cim/route/service/impl/UserInfoCacheServiceImpl.java

```diff
@@ -26,30 +31,16 @@
 @Service
 public class UserInfoCacheServiceImpl implements UserInfoCacheService {
 
-    /**
-     * todo 本地缓存，为了防止内存撑爆，后期可换为 LRU。
-     */
-    private final static Map<Long,CIMUserInfo> USER_INFO_MAP = new ConcurrentHashMap<>(64) ;
-
     @Autowired
     private RedisTemplate<String,String> redisTemplate ;
 
+    @Autowired
+    private LoadingCache<Long, CIMUserInfo> USER_INFO_MAP;
```

Comment:
Please use camel case naming for variable name.

Reply:
Sure, `USER_INFO_MAP` has been replaced by `userInfoMap`.